conans/cli/output.py

```diff
@@ -109,12 +109,19 @@ def rewrite_line(self, line):
         self._color = tmp_color
 
     def _write_message(self, msg, fg=None, bg=None):
+        tmp = ""
         if self._scope:
-            msg = "{}: {}".format(self.scope, msg)
-        if self._color:
-            msg = "{}{}{}{}".format(fg or '', bg or '', msg, Style.RESET_ALL)
+            if self._color:
+                tmp = "{}{}{}:{} ".format(fg or '', bg or '', self.scope, Style.RESET_ALL)
```

Comment:
If we want to simplify output coloring, we can do it too (now the key is that only the ``pkg/version:`` part of the message is colored, while the rest is plain white. But we can change that if we want.